Subdivision_method_3/doc/Subdivision_method_3/Subdivision_method_3.txt

```diff
@@ -552,8 +559,8 @@ namespace Subdivision_method_3 {
 
 \section Subdivision_method_3Other Example: Customizing a Subdivision Method
 
-`Subdivision_method_3` supports four practical subdivision methods on a
-`Polyhedron_3` with points with %Cartesian coordinates. More subdivision methods can be supported
+`Subdivision_method_3` supports four practical subdivision methods on a polyhedral mesh
```

Comment:
```suggestion
`Subdivision_method_3` supports four practical subdivision methods on a polygon mesh
```
We may have to unify this in the subdivision package.

Reply:
I'd say this and the "should / must" are out of scope for this PR.

Reply:
It is cheap to replace the word "polyhedral"  with " polygonal"  (and not " polygon"  as I originally suggested)  just to match the usage in the Subdivision package. See the first sentence in the [introduction](https://doc.cgal.org/latest/Subdivision_method_3/index.html#sectionSubIntro)